catalog/pipeline/fbc-release/0.8/README.md

```diff
@@ -0,0 +1,57 @@
+# FBC Release Pipeline
+
+FBC Release Pipeline
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the EnterpriseContractPolicy | No | - |
+| fromIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| targetIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| overWriteFromIndex | Boolean indicating if the fromIndex should be overwritten | Yes | "false" |
+| binaryImage | OCP binary image to be baked into the index image | Yes | "" |
+| buildTags | List of additional tags the internal index image copy should be tagged with | Yes | "[]" |
+| addArches | List of arches the index image should be built for | Yes | "[]" |
+| requester | Name of the user that requested the signing, for auditing purposes | No | - |
+| signingConfigMapName | The ConfigMap Name required by the Pipeline | Yes | "hacbs-signing-pipeline-config" |
+| fbcPublishingCredentials | Secret used to publish the built index image  | Yes | "fbc-publishing-pull-secret" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | - |
+| buildTimeoutSeconds | Max seconds to wait until the build finishes | Yes | - |
+
+## Changelog
+
+### Changes since 0.7
+- adds task `publish-index-image`
+- adds parameters `targetIndex` and `fbcPublishingCredentials`
```

Comment:
I see `targetIndex` in `0.7/fbc-release.yaml` it just isn't in the README (or sample and test file). Maybe making it its own bullet point since it isn't new functionality, just improved docs and examples